lib/manager/github-actions/extract.ts

```diff
@@ -47,7 +47,7 @@ export function extractPackageFile(content: string): PackageFile | null {
         versioning: dockerVersioning.id,
         depType: 'action',
         replaceString,
-        autoReplaceStringTemplate: `{{depName}}${path}@{{#if newDigest}}{{newDigest}}{{#if newValue}} # renovate: tag={{newValue}}{{/if}}{{/if}}{{#unless newDigest}}{{newValue}}{{/unless}}`,
+        autoReplaceStringTemplate: `{{depName}}${path}@{{#if newDigest}}{{newDigest}}{{#if newValue}}  # renovate: tag={{newValue}}{{/if}}{{/if}}{{#unless newDigest}}{{newValue}}{{/unless}}`,
```

Comment:
We need to make sure this won't break prettier

Reply:
You are right, Prettier removes the extra whitespace again when ran against the resulting yaml.
I assume that is what you were referring to?

```
➜   cat test.yml 
---
name: Test

on:
  pull_request:

jobs:
  testjob:
    name: Tests prettier
    runs-on: ubuntu-latest
    steps:
      - uses: actions/checkout@ec3a7ce113134d7a93b817d10a8272cb61118579  # renovate: tag=v2.4.0
      - run: echo

➜  prettier --config ./.prettierrc test.yml > test-prettier.yaml

➜  diff test.yml test-prettier.yaml                             
12c12
<       - uses: actions/checkout@ec3a7ce113134d7a93b817d10a8272cb61118579  # renovate: tag=v2.4.0
---
>       - uses: actions/checkout@ec3a7ce113134d7a93b817d10a8272cb61118579 # renovate: tag=v2.4.0
➜

```

Reply:
So we need a new idea, as this will broke all of our own repos, as we use prettier on them.

Reply:
Just to make sure I understand correctly, you are talking about running prettier on the code generated by Renovate _outside_ of Renovate, so in a Github action that checks PRs or something similar?

Or do you mean that Renovate runs prettier against the code it generated before creating a PR?

Reply:
We don't run prettier before creating the PR. But any repo which uses prettier will fail the subsequent PR. Not sure if prettier is more widely used than yamllint but it's definitely got significant use

Reply:
Yes, we run prettier as linter on ci checks, so all renovate PR's would fail with this change.

Maybe as first step we can detect the number of spaces on already pinned action and persist it. So renovate won't change it.

I'm not sure if there is a way for renovate to detect on unpinned actions. I think checking for all prettier and yamllint config files would be too expensive.

Reply:
Agreed. Considering that `yamllint` can be configured for this, while `prettier` cannot, I think we're best to keep things simple as-is. If we could unbreak `yamllint` users without breaking `prettier` users, it would be a different decision

Reply:
I agree, lets keep this is is. I'll close the PR and issue as "won't fix"

I'll also open an issue on the yamllint repo if there is a reason for the default being 2 spaces, most stuff I can find elsewhere on the web seems to be happy with just one..

Reply:
Thanks for the PR though!